components/list/list-item-content.js

```diff
@@ -26,13 +26,21 @@ class ListItemContent extends LitElement {
 				overflow: hidden;
 			}
 
+			.d2l-list-item-content-text-supportingInformation {
+				color: var(--d2l-list-item-content-text-color);
+				margin: 0;
+				margin-top: 0.15rem;
+				overflow: hidden;
+			}
+
 		`];
 	}
 
 	render() {
 		return html`
 			<div class="d2l-list-item-content-text d2l-body-compact"><slot></slot></div>
-			<div class="d2l-list-item-content-text-secondary d2l-body-small"><slot name="secondary"></slot></div>
+			<div class="d2l-list-item-content-text-secondary d2l-body-small"><slot name="supportingInformation"></slot></div>
+			<div class="d2l-list-item-content-text-supportingInformation d2l-body-small"><slot name="secondary"></slot></div>
```

Comment:
I am a little confused here though - the class that is being applied here for the secondary slot is `d2l-list-item-content-text-supportingInformation`.  Similarly above.

Reply:
Yeah so where you see `class="d2l-list-item-content-text-supportingInformation d2l-body-small">` we should in fact have `<slot name="supporting-info">` and not `<slot name="secondary"` like here. We are doing it this way because some projects are using `secondary` instead of `supporting-info` e.g: https://search.d2l.dev/xref/GitHub/Brightspace/d2l.dev/sites/www/dist/index.html?r=1d5644b7#35, which should be `supporting-info`. Basically, design added a new slot between the `Primary` and `Secondary` and changed the slot that was named `secondary` to `supporting info`, and that new slot is now named `secondary`. Now because we want to get our stuff moving and don't want to make this a breaking change, I thought we could do this, and once everyone is ready to switch, we can just switch the slot names here. Lol, I hope that makes sense.

Reply:
But for consistency, I can have the class name match the slot name and when we switch, we'll just switch the class name as well.

Reply:
Oh yikes.  We _really_ try to avoid changing slot names because they are part of the public API.  If we need to change the slot name we should really be adding the new slot, going to all the apps that are using the old slot name and update them to the new slot name.  I definitely would not want to see this class/slot naming for very long, since it's going to confuse almost everyone who comes across it, and will likely be subject to being "corrected".

Reply:
Oh yeah, I completely agree, that was the initial plan but then we didn't want to disturbed other teams and have them test their apps again. But I'll bring up your concern to the team and see if we can reconsider going and updating the apps directly. FYI @zinabat @m6jones

Reply:
We discussed various approaches to this, but I didn't think about the possibility of it being "corrected" by others if it stays for too long with this approach. We should consider adding the new slot and do as Dave suggests

Reply:
Yeah just spoke with Jenny as well.  Let's try to get this & and apps set up the way we want so we don't have any debt to clean up.  If you need help updating the various places that depend on this, with approvals or what-not, I can probably give a hand.  It should just be a matter of updating to latest core version, updating to the new slot name, and releasing for new bsi.

Reply:
Cool thanks! Once this is in, I'll start opening PR in the repos that I listed above.